app.cr

```diff
@@ -14,10 +14,26 @@ def headers
   headers
 end
 
-def crystal_repos(sort)
+def crystal_repos
   client = HTTP::Client.new("api.github.com", 443, true)
-  response = client.get("/search/repositories?q=language:crystal&per_page=100&sort=#{sort}", headers)
-  GithubRepos.from_json(response.body)
+  response = client.get("/search/repositories?q=language:crystal&per_page=100", headers)
+  total_repos = GithubRepos.from_json(response.body).total_count
+  repos = GithubRepos.from_json(response.body)
+
+  max_pages = (total_repos.to_f / 100).round.to_i
+  max_pages.times do |i|
```

Comment:
I have been thinking of this a bit, and was wondering if we should do all of this in one request, because this means that one (unlucky) person will get hit with a big waiting time if the data is not cached. Also, we would be sending a lot of data to the template, that's probably not very useful for everyone.

What about adding some sort of pagination, and make subsequent requests only if someone tries to visit the second, third or more pages? That would improve our first page response time, but also provide all the info for someone that really wants to see it.